rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -767,6 +767,15 @@ private long computeDataCopyGas() {
         return calcMemGas(oldMemSize, newMemSize, copySize);
     }
 
+    private long computeMemoryCopyGas() {
+        DataWord length = stack.get(stack.size() - 3);
+        DataWord offset = stack.peek();
+        long copySize = Program.limitToMaxLong(length);
+        checkSizeArgument(copySize);
+        long newMemSize = memNeeded(offset, copySize);
+        return calcMemGas(oldMemSize, newMemSize, copySize) + GasCost.MEMORY;
```

Comment:
could we use `GasCost.add()` instead of plus here?

Reply:
Yup! missed that! thanks!